.github/wiki/Project-Dependencies.md

```diff
@@ -0,0 +1,62 @@
+To simplify setup for developers new to the application, as well as having a reliable build system that is also able to run past builds again in a reproducible fashion. The creation and set up of a new project will be faster and safer requiring only the language runtime (`Ruby`) and dependency manager installed as prerequisites.
```

Comment:
```suggestion
To simplify setup for developers new to the application, as well as having a reliable build system that is also able to run in a reproducible fashion. The creation and set up of a new project will be faster and safer. It will require only the language runtime (`Ruby`) and dependency manager installed as prerequisites.
```
I don't think we run past builds again but reproducible 👍  I think it's lengthly as an introduction so I tried to separated the sentences

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/271/commits/9db8df6e06b41151e5d0e99e502d821e1bd8ea48